int32 position_hip
int32 position_knee
